ignite/contrib/handlers/param_scheduler.py

```diff
@@ -284,3 +284,165 @@ def __call__(self, engine, name=None):
                 self._next_scheduler()
 
         return self._scheduler(engine, name)
+
+
+class LRScheduler(ParamScheduler):
+    """A wrapper class to call `torch.optim.lr_scheduler` objects as `ignite` handlers.
+
+    Args:
+        lr_scheduler (subclass of `torch.optim.lr_scheduler._LRScheduler`): The lr_scheduler object to wrap.
+        save_history (bool, optional): whether to log the parameter values
+            (default=False)
+
+    .. code-block:: python
+
+        from ignite.contrib.handlers.param_scheduler import LRScheduler
+        from torch.optim.lr_scheduler import StepLR
+
+        step_scheduler = StepLR(optimizer, step_size=3, gamma=0.1)
+        scheduler = LRScheduler(step_scheduler)
+
+        trainer.add_event_handler(Events.ITERATION_COMPLETED, scheduler)
+    """
+    def __init__(self, lr_scheduler, save_history=False, **kwds):
+
+        self.lr_scheduler = lr_scheduler
+        super(LRScheduler, self).__init__(
+            optimizer=self.lr_scheduler.optimizer,
+            param_name='lr',
+            save_history=save_history
+        )
+
+    def __call__(self, engine, name=None):
+
+        #
+        # Call the pytorch scheduler step method.
+        #
+        self.lr_scheduler.step(engine.state.epoch)
+
+        #
+        # Call the ignite object __call__ method.
+        # Note:
+        # The 'lr' parameter will be updated for the second time but with the same value.
+        #
+        super(LRScheduler, self).__call__(engine, name)
+
+    def get_param(self):
+        """Method to get current optimizer's parameter value
+        """
+        return self.lr_scheduler.get_lr()
```

Comment:
This can be tricky, `lr_scheduler.get_lr()` return a list of values and not a single value.